### PR TITLE
feat: synced time displays

### PR DESCRIPTION
**Pull request by thomasslee97**
This is a work-in-progress PR to sync the ticking of various timers in the UI.

### CODE REVIEW

**Comment by thomasslee97** [comment]
From discussing with Krzysztof: We may need to implement the options `tickResolution` and `dataResolution` to allow us to have an even lower resolution tick of 1s, while still subscribing to high resolution timing data in some cases.

File: meteor/client/ui/RundownView/RundownTiming/withTiming.tsx [hunk 1]
```diff
@@ -6,8 +6,21 @@ import { RundownTimingContext } from '../../../../lib/rundown/rundownTiming'
...
 
+export enum TimingTickResolution {
+	/** Once per second */
+	Synced = 0,
+	Low,
+	High,
+}
+
+export enum TimingDataResolution {
+	Low = 0,
```
Comment (ianshade) [inline meteor/client/ui/RundownView/RundownTiming/withTiming.tsx:17]: I think the naming should be Synced vs High here, to make it clear that with `TimingTickResolution.Low` and `TimingTickResolution.High` you probably should still use `TimingDataResolution.High`, and `TimingTickResolution.Synced` should go with `TimingDataResolution.Synced`

Comment (thomasslee97) [reply meteor/client/ui/RundownView/RundownTiming/withTiming.tsx]: Good idea, I've renamed that https://github.com/tv2/tv-automation-server-core/commit/0ec1c3d8876f2f5cb2915cb4fdd5b22160af181d

**Review by ianshade — APPROVED** [review]

File: meteor/client/ui/RundownView/RundownTiming/withTiming.tsx [hunk 2]
```diff
@@ -102,12 +118,47 @@ export function withTiming<IProps, IState>(
 				// To bring it back to sync, we mark the component as dirty, which will
 				// force an update on the next low resoluton tick, regardless of what
 				// the filter says.
-				if (!!this.filterGetter && durations.isLowResolution !== !expandedOptions.isHighResolution) {
+				if (
```
Comment (skon-tv2) [inline meteor/client/ui/RundownView/RundownTiming/withTiming.tsx:121]: The condition of this if should be moved to a function

File: meteor/client/ui/RundownView/RundownTiming/RundownTiming.ts [hunk 3]
```diff
@@ -22,8 +23,10 @@ export namespace RundownTiming {
 	 * @enum {number}
 	 */
 	export enum Events {
+		/** Event is emitted once a second, to update displays in a synced manner */
+		'timeupdateSynced' = 'sofie:rundownTimeUpdateSynced',
 		/** Event is emitted every now-and-then, generally to be used for simple displays */
-		'timeupdate' = 'sofie:rundownTimeUpdate',
+		'timeupdateLR' = 'sofie:rundownTimeUpdateLR',
```
Comment (skon-tv2) [inline meteor/client/ui/RundownView/RundownTiming/RundownTiming.ts:29]: I would prefer LowResolution instead of just LR and HighResolution instead of HR

**Review by skon-tv2 — APPROVED** [review]

**Pull request merged**